### DISCUSSION TITLE
Wer steht jetzt eigentlich hinter Cruiser

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#636] [Q&A] [answered]
Hallo,

Ich habe jetzt mal ne allgemeine Frage, da ich etwas verwirrt bin. Steht jetzt hinter der Leitung und Entwicklung der Cruiser Software nach wir vor @devemux86 oder jetzt motoukrider.

Hello,

I have a general question as I am a bit confused. Is @devemux86 still behind the management and development of the Cruiser software or now motoukrider.

**Comment by devemux86** [comment, marked as answer]
- I am the author of Cruiser.
- @MotoUKRider helps by writing the Motorider theme: https://github.com/devemux86/cruiser/discussions/612

---

- Ich bin der Autor von Cruiser.
- @MotoUKRider hilft, indem er das Motorider-Theme schreibt: https://github.com/devemux86/cruiser/discussions/612

**Reply by ToToGSARider** [reply]
Ah Okay, ich dachte schon du hättest Cruiser übergeben und würdest dich jetzt langsam zurückziehen. Deine Antwort gestern hat mich stutzig gemacht, als du gesagt hattest wir müssen das mit dem Kontrast an ihn melden. Ist dann dieses Layout automatisch aktiviert oder muss man das bewusst aktualisieren?

Ah okay, I thought you had handed Cruiser over and were now slowly backing off. Your answer yesterday made me wonder when you said we had to report the contrast to him. Is this layout automatically activated or do you have to update it deliberately?

**Reply by devemux86** [reply]
@ToToGSARider 

Motorider theme is updated when Cruiser is updated.

Das Thema Motorider wird aktualisiert, wenn Cruiser aktualisiert wird.
